models/balancer/balancer_v2_liquidity.sql

```diff
@@ -0,0 +1,41 @@
+{{ config(
+        alias ='liquidity', 
+        post_hook='{{ expose_spells(\'["ethereum","arbitrum", "optimism", "polygon", "gnosis","avalanche_c" 
+        ]\',
+                                "project",
+                                "balancer",
+                                \'["viniabussafi"]\') }}'
+        )
+}}
+
+{% set balancer_models = [
+ref('balancer_v2_ethereum_liquidity')
+, ref('balancer_v2_optimism_liquidity')
+, ref('balancer_v2_arbitrum_liquidity')
+, ref('balancer_v2_polygon_liquidity')
+, ref('balancer_v2_gnosis_liquidity')
+, ref('balancer_v2_avalanche_c_liquidity')
+] %}
+
+
+SELECT *
+FROM (
+    {% for flash_model in balancer_models %}
```

Comment:
You can use any name for the x in `{% for x in balancer_models %}`. Maybe `liquidity_model` or something similar would be good.